### ISSUE TITLE
request kotatogram-desktop-bin

### ISSUE THREAD

**Issue by robson-66** [chaotic-aur/packages#721]
Please add `kotatogram-desktop-bin`
 https://aur.archlinux.org/packages/kotatogram-desktop-bin/

**Comment by xiota** [comment]
See #1565

**Comment by Technetium1** [comment]
@xiota I may re-add this someday in the future when Kotatogram gets updated, unless `kotatogram-desktop-beta-dynamic-bin` remains functional at that point. I used this as a fallback.

**Comment by xiota** [comment]
@Technetium1  This issue was just closed as a duplicate of #1565.  Package is still in lists.
```
$ grep -Rin kotato
garuda-cluster/hourly.1.txt:250:kotatogram-dev-git
ufscar-hpc/hourly.1.txt:856:kotatogram-desktop-beta-dynamic-bin
ufscar-hpc/hourly.1.txt:1025:kotatogram-desktop-bin

$ pacman -Ss kotato
chaotic-aur/kotatogram-desktop-beta-dynamic-bin 1.4.9-5.2
    Kotatogram – experimental Telegram Desktop fork - Beta version precompiled package
chaotic-aur/kotatogram-desktop-bin 1.4.9-1
    Experimental Telegram Desktop fork with option to select custom fonts. - Static binaries
chaotic-aur/kotatogram-dev-git k1.4.9.r2.g52b300cb3-1
    Kotatogram – experimental Telegram Desktop fork - Dev branch
```
